output/outas86.c

```diff
@@ -110,6 +110,8 @@ static void as86_sect_write(struct Section *, const uint8_t *,
 
 static void as86_init(void)
 {
+    char filename[FILENAME_MAX];
```

Comment:
We probably don't want this to be on the stack.

Reply:
Moving this out of the stack would be an excellent thing. Depending on MAX_PATH, FILENAME_MAX et al is highly discouraged, too (I would like to make that the empty set.)